examples/asr/speech_pretraining/README.md

```diff
@@ -15,6 +15,15 @@ For their corresponding usage, please refer to the example yaml config:
 - NEST: `examples/asr/conf/ssl/nest/nest_fast-conformer.yaml`
```

Comment:
Its also better to add these sample manifest lines to training config file: examples/asr/conf/ssl/nest/nest_fast-conformer.yaml